monkestation/code/modules/smithing/oozelings/body/organs.dm

```diff
@@ -69,8 +69,15 @@
 	var/rebuilt = TRUE
 	var/coredeath = TRUE
 
+	var/datum/action/cooldown/membrane_murmur/membrane_mur
+
+/obj/item/organ/internal/brain/slime/Destroy(force)
```

Comment:
bit of a nitpick, but can you move the code Destroy below the Initialize code?

otherwise, looks good.

Reply:
There we go resolved.